frontend/src/pages/application/Snapshots/components/dialogs/SnapshotExportDialog.vue

```diff
@@ -1,29 +1,32 @@
 <template>
     <ff-dialog ref="dialog" header="Download Snapshot" confirm-label="Download" :closeOnConfirm="false" :disable-primary="!formValid" data-el="snapshot-download-dialog" @confirm="confirm()">
         <template #default>
-            <form class="space-y-6 mt-2" data-form="snapshot-export" @submit.prevent>
-                <p>Please make a note of the secret used to encrypt the snapshot credentials. It will be required when importing the snapshot.</p>
-                <FormRow containerClass="w-auto" :error="errors.secret" data-form="snapshot-secret">
-                    Secret
-                    <template #description>
-                        <p class="text-sm">A key used to encrypt any credentials in the snapshot.</p>
-                    </template>
-                    <template #input>
-                        <div class="flex items-center w-full">
-                            <ff-text-input ref="secret" v-model="input.secret" type="text" placeholder="Secret" />
-                            <ff-button v-ff-tooltip:top="'Random Secret'" kind="secondary" size="small" class="ml-2" data-el="refresh" @click="input.secret = generateRandomKey()">
-                                <template #icon>
-                                    <RefreshIcon />
-                                </template>
-                            </ff-button>
-                            <ff-button v-if="clipboardSupported" v-ff-tooltip:top="'Copy to Clipboard'" kind="secondary" size="small" class="ml-2" @click="copySecret()">
-                                <template #icon>
-                                    <ClipboardCopyIcon />
-                                </template>
-                            </ff-button>
-                        </div>
-                    </template>
-                </FormRow>
+            <form data-form="snapshot-export" @submit.prevent>
+                <ExportInstanceComponents v-model="parts" :error="errors.parts" header="Select the components to include in the snapshot" />
+                <template v-if="needsSecret">
+                    <FormRow containerClass="w-auto mt-6" :error="errors.secret" data-form="snapshot-secret">
+                        Secret
+                        <template #description>
+                            <p class="text-sm">A key used to encrypt any credentials in the snapshot.</p>
```

Comment:
```suggestion
                            <p class="text-sm">A key used to encrypt any credentials in the snapshot's flow.</p>
```

Fairly sure I'm right in saying that the `credentials` are in the `flow.json`? As an inexperienced NR user, I wouldn't necessarily know the difference here between env vars and credentials, I'd think credentials could be stored in Env Vrs. So want to add a little clarity here.